bin/webpack.js

```diff
@@ -29,35 +29,59 @@ try {
 	webpackCliInstalled = false;
 }
 
+let webpackCommandInstalled = false;
+try {
+	require.resolve("webpack-command");
+	webpackCommandInstalled = true;
+} catch (err) {
+	webpackCommandInstalled = false;
+}
+
 if (!webpackCliInstalled) {
 	const path = require("path");
 	const fs = require("fs");
 	const readLine = require("readline");
+
+	console.error(
+		"The CLI for webpack must be installed as a separate package, for which there are two choices:\n" +
+			"    webpack-cli (https://github.com/webpack/webpack-cli) : The original webpack CLI from webpack@3.\n" +
```

Comment:
Feed this information from the array. Maybe put objects like this into the array:

``` js
[{
  package. "webpack-cli",
  url: "https://github.com/webpack/webpack-cli",
  description: "The full-featured CLI."
}, ...]
```